ext/date/lib/tm2unixtime.c

```diff
@@ -334,9 +334,9 @@ static timelib_sll do_years(timelib_sll year)
 static timelib_sll do_months(timelib_ull month, timelib_ull year)
 {
 	if (timelib_is_leap(year)) {
-		return ((month_tab_leap[month - 1] + 1) * SECS_PER_DAY);
+		return (((timelib_ull)month_tab_leap[month - 1] + 1) * SECS_PER_DAY);
```

Comment:
Not sure what the point is here - month_tab_leap is a set of small constants, what kind of overflow may they cause?